azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CAuthenticationProperties.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.b2c;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.URL;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Setter
+@Validated
+@ConfigurationProperties(prefix = AADB2CAuthenticationProperties.PREFIX)
+public class AADB2CAuthenticationProperties {
+
+    public static final String PREFIX = "azure.activedirectory.b2c";
+
+    private static final String POLICIES = "policies";
+
+    private static final String SIGN_UP_OR_SIGN_IN = "sign-up-or-sign-in";
```

Comment:
Do we really need to distinguish sign in or up here?

Reply:
Yes, the naming is from portal, as there are sign-in and sign-up policies.